server.js

```diff
@@ -49,6 +53,19 @@ function authenticate(code, cb) {
   req.on('error', function(e) { cb(e.message); });
 }
 
+function log(label, value, sanitized) {
+  value = value || '';
+  if (sanitized){
+    if (typeof(value) === 'string' && value.length > 10){
+      console.log(label, value.substring(3,0) + '...');
+    } else {
+      console.log(label, '...');
```

Comment:
Replace hard-coded `’...’` string with a named constant. This will make it easy to change the displayed string everywhere in the future without having to hunt through the code for each hard-coded instance.